README.md

```diff
@@ -1,5 +1,14 @@
 # iroh
 
+Iroh is a next-generation implementation the Interplanetary File System ([IPFS](https://ipfs.io)) for Cloud & Mobile platforms. IPFS is a networking protocol for exchanging _content-addressed_ blocks of immutable data. “content-addressed” means referring to data by the *hash of it’s content*, which makes the reference unique and verifiable. These two properties make it possible to get data from *any* node in the network that speaks the IPFS protocol, including IPFS content being served by other implementations of the protocol.
+
+Iroh is built & maintained by [number 0](https://n0.computer)
+
+## Contributing
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
```

Comment:
can we keep this under license, `contributing` is much more about code/tests etc

Reply:
Sure